src/apps/filters/h3.c

```diff
@@ -110,6 +113,38 @@ bool latLngToCellCmd(int argc, char *argv[]) {
     return true;
 }
 
+bool cellToBoundaryCmd(int argc, char *argv[]) {
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .required = true,
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
+    };
+    Arg helpArg = ARG_HELP;
+    DEFINE_CELL_ARG(cell, cellArg);
+    Arg *args[] = {&cellToBoundaryArg, &helpArg, &cellArg};
+    if (parseArgs(argc, argv, 3, args, &helpArg,
+                  "Convert an H3 cell to an array of latitude/longitude "
+                  "coordinates defining its boundary")) {
+        return helpArg.found;
+    }
+    CellBoundary cb;
+    int err = H3_EXPORT(cellToBoundary)(cell, &cb);
+    if (err) {
+        return false;
+    }
+    // TODO: Is it better to use brackets or parentheses for the output?
+    printf("[\n");
+    for (int i = 0; i < cb.numVerts; i++) {
+        LatLng *ll = &cb.verts[i];
+        printf("  [%.10lf, %.10lf],\n", H3_EXPORT(radsToDegs)(ll->lat),
+               H3_EXPORT(radsToDegs)(ll->lng));
+    }
+    printf("]\n");
+    return true;
```

Comment:
it occurs to me we may want to have different forms of the output, and particularly we may want to support WKT encoding